coderd/database/queries/workspaces.sql

```diff
@@ -287,6 +287,12 @@ WHERE
 	AND LOWER("name") = LOWER(@name)
 ORDER BY created_at DESC;
 
+-- name: GetWorkspaceUniqueOwnerCountByTemplateIDs :many
+SELECT template_id, COUNT(DISTINCT owner_id) AS unique_owners_sum
+FROM workspaces
+WHERE template_id = ANY(@template_ids :: uuid[]) AND deleted = false
+GROUP BY template_id;
+
```

Comment:
Not that it really matters, but most of our sqlc is formatted like:
```sql
-- name: GetWorkspaceUniqueOwnerCountByTemplateIDs :many
SELECT 
	template_id, COUNT(DISTINCT owner_id) AS unique_owners_sum
FROM 
	workspaces
WHERE 
	template_id = ANY(@template_ids :: uuid[]) AND deleted = false
GROUP BY template_id;
```

Reply:
I'm guessing we do not care the state of the workspace besides deleted? Like stopped vs failed etc?